front/app/components/Form/Components/Layouts/CLSurveyPageLayout.tsx

```diff
@@ -242,75 +206,48 @@ const CLSurveyPageLayout = memo(
       scrollToTop();
     };
 
-    if (!project) {
-      return null;
-    }
-
     return (
       <>
-        <Box
-          width="100%"
-          height="100%"
-          pt="82px"
-          pb="72px"
-          maxWidth="700px"
-          display="flex"
-          flexDirection="column"
-          justifyContent="center"
-          alignItems="center"
-        >
-          <SurveyHeading
-            project={project.data}
-            titleText={localize(
-              phase?.data.attributes.native_survey_title_multiloc
-            )}
-            canUserEditProject={userIsModerator}
-            loggedIn={!isNilOrError(authUser)}
-            percentageAnswered={percentageAnswered}
-          />
-
+        <Box display="flex" flexDirection="column" height="100%">
           {allowAnonymousPosting && (
             <Box w="100%" px={isSmallerThanPhone ? '16px' : '24px'} mt="16px">
               <Warning icon="shield-checkered">
                 {formatMessage(messages.anonymousSurveyMessage)}
               </Warning>
             </Box>
           )}
-          <Box
-            display="flex"
-            flex="1"
-            height="100%"
-            pt={isSmallerThanPhone ? '28px' : '8px'}
-            overflowY="auto"
-            w="100%"
-            ref={pagesRef}
-          >
+          <Box h="100%" display="flex" ref={pagesRef}>
             {uiPages.map((page, index) => {
               const pageElements = extractElementsByOtherOptionLogic(
                 page,
                 data
               );
               return (
                 currentStep === index && (
-                  <StyledFormSection key={index}>
-                    <Box
-                      display="flex"
-                      justifyContent="center"
-                      h={hasScrollBars ? 'fit-content' : '100%'}
```

Comment:
I don't think this was making any difference. Tested various fields. But let me know if I missed something!